src/stories/containers/users/user-profile/user-profile.tsx

```diff
@@ -35,6 +35,13 @@ const UserProfile = () => {
   }, [checked, clientRequest]);
 
   const handleDeleteAccount = useCallback(() => {
+    router.push({
+      pathname: '/auth/delete-account/',
+      query: {
+        userName: user?.username,
+        id: user?.id,
```

Comment:
You're expecting here that the `user` can be empty/undefined, but if that is possible you should handle that somehow or you will get an error in the `delete-account` page due the user is undefined

Reply:
The way that is handle, is that the user doesn't exist, then will show a error when you try to delete the user